### PR TITLE
fix(mria): Update description of core-replicant configuration

### PR DESCRIPTION
**Pull request by ieQu1**

### CODE REVIEW

**Review by zmstone — DISMISSED** [review]

**Comment by Meggielqk** [comment]
@ieQu1, I think it should be more specific about "limited support" in the following sentence. Why `manual` strategy is not used and what are the limitations?

"Given that replicant nodes are designed to be stateless, they have limited support for the `manual` discovery strategy."

**Comment by ieQu1** [comment]
@Meggielqk `manual` strategy in EMQX works by joining the database schema of EMQX nodes, which is what replicants were specifically designed to **avoid**.
Originally, `manual` was not allowed on the replicants at all, and was only added by request of QA team, who needed it for unrelated tests. It works by *emulating* some features of the conjoined DB schema, just enough to run test scenarios. It's only meant for testing.
Hence "limited".

**Review by thalesmg — DISMISSED** [review]

File: en_US/deploy/cluster/mria-introduction.md [hunk 1]
```diff
@@ -2,30 +2,31 @@
...
+By default, all nodes assume the Core node role, so the cluster behaves like that in [EMQX 4.x](https://docs.emqx.com/en/enterprise/v4.4/getting-started/cluster.html#node-discovery-and-autocluster), which is recommended for a small cluster with 3 nodes or fewer. The Core + Replicant mode is only recommended if there are more than 3 nodes in the cluster.
 
 ## Enable Core + Replicant Mode
 
-To enable the  Core + Replicant mode, the backend database (`db_backend`) should be set to `rlog`, some nodes should assume the replicant role (`node.db_role`), and the core node (`core_node`) should be specified, as shown below:
+To enable the Core + Replicant mode, it is necessary to designate certain nodes as replicant nodes. This is achieved by setting `node.role` parameter to `replicant`. Additionally, you need to enable an automatic cluster discovery strategy (`cluster.discovery_strategy`). Given that replicant nodes are designed to be stateless, they have limited support for the `manual` discovery strategy.
```
Comment (zmstone) [inline en_US/deploy/cluster/mria-introduction.md]:
```suggestion
To enable the Core + Replicant mode, it is necessary to designate certain nodes as replicant nodes. This is achieved by setting `node.role` parameter to `replicant`. Additionally, you need to enable an automatic cluster discovery strategy (`cluster.discovery_strategy`). 

:::tip
Replicant nodes cannot use `manual` discovery strategy to discover core nodes. 
:::
```

File: zh_CN/deploy/cluster/mria-introduction.md [hunk 2]
```diff
@@ -60,6 +58,24 @@ Mria 是 Mnesia 的一个开源扩展，它为集群增加了最终的一致性
 - 在超大的集群中（10 个节点或更多），核心节点不承载 MQTT 连接，这样更加稳定性和水平扩展性更好。
 - 在中型集群中，核心节点是否承载 MQTT 连接取决于许多因素，需要根据您实际的场景测试才能知道哪个选择更优。
 
+### 启用 Core + Replicant 模式
+
+为了启用 Core + Replicant 模式，需要将某些节点指定为复制节点。这可以通过设置 `node.role` 参数为 `replicant` 来实现。此外，您需要启用一个自动集群发现策略（`cluster.discovery_strategy`）。鉴于复制节点被设计为无状态，它们仅有限支持 `manual` 发现策略。
```
Comment (zmstone) [inline zh_CN/deploy/cluster/mria-introduction.md]:
```suggestion
为了启用 Core + Replicant 模式，需要将某些节点指定为复制节点。这可以通过设置 `node.role` 参数为 `replicant` 来实现。此外，您需要启用一个自动集群发现策略（`cluster.discovery_strategy`）。

:::tip
Replicant 节点不能使用 `manual` 策略来发现 Core 节点集群。
:::
```

**Review by zmstone — APPROVED** [review]

**Pull request merged**